wordpress-mobile/WordPress-iOS
# Migrated SharingServiceRemote to WPKit

Fixes #7328  

**Models:**
* Migrates the `RemoteSharingButton.swift` model to the WPKit framework
* Migrates the `KeyringConnection.swift` model to the WPKit framework
* Migrates the `KeyringConnectionExternalUser.swift` model to the WPKit framework
* Migrates the `RemotePublicizeConnection.swift` model to the WPKit framework
* Migrates the `RemotePublicizeService.swift` model to the WPKit framework

**Remotes:**
* Migrates the `SharingServiceRemote.swift`into the WPKit framework
* Migrates the `SharingServiceRemoteTests.m` test into the WPKit framework

**WPShared:**
* Migrates `DateUtils.h/.m` to the WPShared framework

**Misc:**
* Updates related imports/refs in main WordPress project

Ref #7301

**To test:**
1. Clean DerivedData.
2. Update CocoaPods repo with pod repo update.
3. Install the pods with pod install.
4. Run _all_ unit tests and make sure they are green.
5. Build & run the app ... verify the sharing section is working as expected:

<img width="192" alt="napkin 43 07-18-17 12 43 25 pm" src="https://user-images.githubusercontent.com/154014/28331451-b9a0316e-6bb6-11e7-8b09-d068135b855a.png">

@astralbodies would you mind reviewing this?
